test/comments.d

```diff
@@ -232,7 +234,11 @@ unittest
 8573,"A simpler Phobos function that returns the index of the mix or max item","NEW","---","regression","P2"`);
         },
         "/github/orgs/dlang/public_members?per_page=100",
-        // no bug fix label, since Issues are only referenced but not fixed according to commit messages
+        "/github/repos/dlang/phobos/issues/4921/labels",
+        "/github/repos/dlang/phobos/issues/4921/labels",
```

Comment:
The bot tries to add a "Bug fix" label now, so unfortunately we have to be a bit explicit